### PR TITLE
instance_profiling: refine, remove Math.random() as key

### PR DESCRIPTION
**Pull request by baurine**
What did:

- A small refine, remove `Math.random()` as key for instance profiling pages

### CODE REVIEW

File: ui/lib/apps/InstanceProfiling/index.tsx [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+  Routes,
+  useParams,
+} from 'react-router-dom'
 
 import { Root } from '@lib/components'
+import { Detail, List } from './pages'
 
-import DetailPage from './pages/Detail'
-import ListPage from './pages/List'
+function DetailPageWrapper() {
```
Comment (breeswish) [inline ui/lib/apps/InstanceProfiling/index.tsx:12]: Maybe this function can be a common pattern :) I believe for all pages using `useClientRequest` we'd better use a similar solution.

Comment (baurine) [reply ui/lib/apps/InstanceProfiling/index.tsx:12]: Make sense, let me try it.

File: ui/lib/components/ParamPageWrapper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+export interface IParamPageWrapperProps {
+  paramName?: string
+  children: ReactNode
+}
+
+export default function ParamPageWrapper({
+  paramName = 'id',
+  children,
+}: IParamPageWrapperProps) {
+  const paramValue = useParams()[paramName]
```
Comment (breeswish) [inline ui/lib/components/ParamPageWrapper/index.tsx:13]: How about simply use the JSON stringify value of `useParams()` as the key? Then no need to specify the param name.

Comment (baurine) [reply ui/lib/components/ParamPageWrapper/index.tsx:13]: Good idea! Let me try it.

Comment (baurine) [reply ui/lib/components/ParamPageWrapper/index.tsx:13]: submit PR in #527

**Pull request merged**